packages/gestalt-codemods/0.33.0-0.34.0/avatar-non-responsive.js

```diff
@@ -56,7 +56,8 @@ export default function transformer(file, api) {
             if (convertSizes[value]) {
               attribute.value.value = convertSizes[value];
               return attribute;
-            } else if (invalidSizes.includes(value)) {
+            }
+            if (invalidSizes.includes(value)) {
```

Comment:
wow huh

Reply:
@chrislloyd same reaction haha, its this rule https://eslint.org/docs/rules/no-else-return which does not like an `else if` clause after a `return`